packages/react-widgets/src/widgets/CategoryWidget.js

```diff
@@ -113,12 +111,12 @@ function CategoryWidget(props) {
   );
 
   return (
-    <WrapperWidgetUI title={title} isLoading={widgetsLoadingState[id]} {...wrapperProps}>
+    <WrapperWidgetUI title={title} isLoading={isLoading} {...wrapperProps}>
       <CategoryWidgetUI
         data={categoryData}
         formatter={formatter}
         labels={labels}
-        isLoading={widgetsLoadingState[id]}
+        isLoading={isLoading}
```

Comment:
Why is it here the isLoading preserved (vs other widgets)? Does it really have an effect on the widget or could we just leave the generic isLoading for WrapperWidgetUI ?

Reply:
Yes, CategoryWidgetUI uses isLoading prop for its logic. It was there before my changes.